packages/custom-elements/ts_src/Patch/Node.ts

```diff
@@ -92,9 +92,14 @@ export default function (internals: CustomElementInternals) {
 
   Node.prototype.cloneNode = function (this: Node, deep) {
     const clone = Native.Node_cloneNode.call(this, !!deep);
+    // document.ownerDocument is null
+    const registry = this.ownerDocument
+      ? this.ownerDocument!.__CE_registry
+      : (this as Document).__CE_registry;
```

Comment:
I think this could be simplified to `(this.ownerDocument ?? this).__CE_registry` and kept inline.

Reply:
Done, thanks